build.gradle

```diff
@@ -17,65 +17,61 @@ version = release_version
 sourceCompatibility = JavaVersion.VERSION_11
 targetCompatibility = JavaVersion.VERSION_11
 
-ext {
-    grpcVersion         = '1.48.1'
-    protobufVersion     = '3.21.12'
-    sharedDir           = file("${project.rootDir}/shared")
-}
-
 repositories {
     mavenCentral()
     maven {
-            name 'Sonatype_snapshots'
-            url 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
+        name 'Sonatype_snapshots'
+        url 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
     }
     maven {
-            name 'Sonatype_releases'
-            url 'https://s01.oss.sonatype.org/content/repositories/releases/'
+        name 'Sonatype_releases'
+        url 'https://s01.oss.sonatype.org/content/repositories/releases/'
     }
 
-    configurations.all {
+    configurations.configureEach {
         resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
         resolutionStrategy.cacheDynamicVersionsFor 0, 'seconds'
     }
 }
 
 dependencies {
-    api platform('com.exactpro.th2:bom:4.2.0')
+    api platform('com.exactpro.th2:bom:4.4.0-update-libs-5345334944-c71fdc9-SNAPSHOT')
 
     // Grpc and Protobuf
-    implementation "com.google.protobuf:protobuf-java:${protobufVersion}"
     api "com.google.protobuf:protobuf-java-util"
     api "io.grpc:grpc-stub"
-    
+
+    implementation "com.google.protobuf:protobuf-java"
+
     implementation "org.slf4j:slf4j-api"
-    
-    //Kotlin
-    implementation "org.jetbrains.kotlin:kotlin-stdlib"
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
 
-    implementation 'io.github.microutils:kotlin-logging:2.0.4'
+    //Kotlin
+    implementation 'io.github.microutils:kotlin-logging:3.0.0' // The last version bases on kotlin 1.6.0
 
     //For generation java code. License: Apache 2.0
     implementation "com.squareup:javapoet:1.13.0"
-    
-    implementation "commons-io:commons-io:2.11.0"
 
-    testCompile group: 'junit', name: 'junit', version: '4.12'
+    implementation "commons-io:commons-io"
+
+    testImplementation "org.junit.jupiter:junit-jupiter:5.8.2"
```

Comment:
The latest version is [5.9.3](https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-api/5.9.3). I think we can use it instead 5.8.2